apps/datahub/src/app/record/record-metadata/record-metadata.component.html

```diff
@@ -120,7 +120,35 @@
     </div>
 
     <div *ngIf="displayApi$ | async">
-      <datahub-record-apis class="block mt-6"></datahub-record-apis>
+      <datahub-record-apis
+        [isPanelOpen]="isPanelOpen"
+        (selectedLink)="selectApiLink($event)"
+        class="block mt-6"
+      ></datahub-record-apis>
+    </div>
+  </div>
+  <div
```

Comment:
I think that this section should be within the `datahub-record-apis` component, it's purely related.
WSYT @tkohr ?

Reply:
Ah right, I missed that too. Ideally, I'd see the lines I commented below in the `record-api-form.component` and then the `record-api-form.component` go into the `record-metadata.comonent`. WDYT @cmoinier @fgravin ?